README.md

```diff
@@ -14,13 +14,20 @@ Add this to the top of your `Makefile`:-
 include n.Makefile
 ```
 
+If you plan to use `make build-production` then you will need to add `haikro` as a dev dependency in your project:-
```

Comment:
=> ‘If you're app has a `Procfile` then you will need to add `haikro` as a dev dependency……’